sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -141,7 +141,7 @@ Version 7.0.0b1 is a preview of our efforts to create a client library that is u
 
 * Introduces new AMQP-based API.
 * Original HTTP-based API still available under new namespace: azure.servicebus.control_client
-* For full API changes, please see updated [reference documentation](https://docs.microsoft.com/python/api/overview/azure/servicebus/client?view=azure-python).
+* For full API changes, please see updated [reference documentation](https://docs.microsoft.com/en-us/python/api/azure-servicebus/azure.servicebus?view=azure-python).
```

Comment:
I think it's better to remove en-us/ from the link

Reply:
I am doing that in a separate PR

Reply:
```suggestion
* For full API changes, please see updated [reference documentation](https://docs.microsoft.com/python/api/azure-servicebus/azure.servicebus?view=azure-python).
```